packages/esm-ward-app/src/types/index.ts

```diff
@@ -161,3 +163,24 @@ export interface EncounterRole extends OpenmrsResourceStrict {
   description?: string;
   retired?: boolean;
 }
+
+export interface PatientNotePayload {
```

Comment:
This can just be called "EncounterPayload".... ideally it would be a type already defined in ESM Core somewhere, but if not, fine to define it here. (fyi @ibacher @denniskigen )

Reply:
let's just go ahead then and rename this EncounterPayload